linux/lvm.go

```diff
@@ -314,12 +376,7 @@ func (d *lvmLVData) toLV() disko.LV {
 		lvtype = disko.THIN
 	}
 
-	if pathExists(d.Path) {
-		_, _, rc := runCommandWithOutputErrorRc("cryptsetup", "isLuks", d.Path)
-		if rc == 0 {
-			crypt = true
-		}
-	}
+	crypt, cryptName, _ := getLuksInfo(d.Path)
```

Comment:
Do you really want to ignore this error?

Reply:
well... i did, but you're right. i probably shouldn't.

I have changed to pass it up the chain now.